CI_YAML.md

```diff
@@ -80,7 +80,7 @@ targets:
 
 #
 # Devicelab example:
-# For tests that are located https://github.com/flutter/flutter/tree/master/dev/devicelab/bin/tasks:
+# For tests that are located https://github.com/flutter/flutter/tree/main/dev/devicelab/bin/tasks:
```

Comment:
The default branch for flutter/flutter is master, not main.

Reply:
I'm anticipating the imminent (?) switch to `main` and future deprecation of `master`, according to https://github.com/flutter/flutter/issues/121564. `main` copies commits from `master` so all links are valid